tools/release/create-pr-to-merge-release-branch-to-main.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: AGPL-3.0-only
+
+set -e
+
+# Check input parameters.
+if [ $# -gt 1 ]; then
+  echo "Usage: $0 [release-branch]"
+  echo ""
+  echo "  release-branch  The release branch name. If omitted, the latest"
+  echo "                  release branch will be auto-detected."
+  echo ""
+  exit 1
+elif [ $# -eq 1 ]; then
+  LAST_RELEASE_BRANCH="$1"
+fi
+
+# Load common lib.
+CURR_DIR="$(dirname "$0")"
+. "${CURR_DIR}/common.sh"
+
+check_required_tools
+
+# Ensure there are not uncommitted changes.
+if [ -n "$(git status --porcelain=v1 2> /dev/null)" ]; then
+  echo "Please make sure there are no uncommitted changes."
+  exit 1
+fi
+
+# Auto-detect the last release branch, if it wasn't provided.
+if [ -z "${LAST_RELEASE_BRANCH}" ]; then
+  LAST_RELEASE_BRANCH=$(git branch --list 'release-*' | sort -V | grep -Eo 'release-[^ ]+' | tail -1)
+  if [ -z "${LAST_RELEASE_BRANCH}" ]; then
+    echo "Unable to find the last release branch. You can provide it directly to the script."
+    exit 1
+  fi
+fi
+
+BRANCH_NAME="merge-${LAST_RELEASE_BRANCH}-to-main"
+
+# Ask confirmation.
+read -p "You're about to open a PR to merge '${LAST_RELEASE_BRANCH}' to main. Do you want to continue? (y/n) " -n 1 -r
+echo ""
+if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+    echo "Aborting ..."
+    exit 1
+fi
+
+# Prepare the branch.
+git checkout main
+git pull
+git checkout -b "${BRANCH_NAME}"
+
+# The merge command could fail if there are conflicts but we don't want to abort the script in that case.
+echo ""
+git merge "${LAST_RELEASE_BRANCH}" || true
+echo ""
+
+# Wait until the status is clean.
+while true; do
+  # Print some instructions on how to proceed.
+  echo "The '${LAST_RELEASE_BRANCH}' branch has been merged to main. Please do the following:"
+  echo "1. Ensure VERSION is not changed (keep the same version as in main)"
+  echo "2. Fix any conflict and run 'git commit' once done"
```

Comment:
To conclude merge with conflicts, one should use `git merge --continue`

Reply:
> To conclude merge with conflicts, one should use `git merge --continue`

Man page says `git commit` is fine too...

> Use git commit or git merge --continue to seal the deal. The latter command checks whether there is a (interrupted) merge in progress before calling git commit.

(from https://git-scm.com/docs/git-merge)